### ISSUE TITLE
Splat not using terrain obstruction in computed dBm levels, making huge error

### ISSUE THREAD

**Issue by meresmclr** [watkipet/splat#2]
When using Splat in dBm output mode, that is, where transmitter ERP is specified and dBm is the desired quantity, I notice that the terrain obstruction loss is not being used, the final predicted dBm is based on ITWOM 3 only as if it were a line-of-sight link, making 10's of dB error. I tested only the "Double Horizon Diffraction Dominant" of Splat! vs. Radio Mobile Deluxe.

This is true of Splat 1.4.0 as well as this development version.

Separately, glad you have overhauled the manual bash install, Cmake was sorely needed. Planning to make minor tweak to allow non-root install to ~/.local optional.

**Comment by watkipet** [comment]
Thanks! I'm still learning git and github (the reason I forked Splat! was to learn). Once you've made your changes, how would you like to send them to me? Pull request?

Just to confirm, this is the type of analysis you're running, right?

```
RECEIVED POWER LEVEL ANALYSIS
   If the transmitter's effective radiated power (ERP) is specified in the
   transmitter's  .lrp  file,  or  expressed on the command-line using the
   -erp switch, and the -dbm switch is invoked, received power level  con-
   tours referenced to decibels over one milliwatt (dBm) are produced:

   splat -t wnjt-dt -L 30.0 -erp 46000 -dbm -db -100 -o plot.ppm
```

**Comment by meresmclr** [comment]
Yes that's right. So perhaps I should kick it up to the original author, as this appears to be a severely misleading output from the program.

**Comment by watkipet** [comment]
Yeah, I think kicking it upstream is best. I think the original author will
want to fix this before merging my changes, and merging the fix from my
source is going to be a pain. I might be able to help with finding root
cause though.

**Comment by meresmclr** [comment]
Hmm, now I found that when I run a case like that on the author's webpage for WNYE, my results are similar. So perhaps I was making some other mistake, or I had a case where the end user should be warned.

Separately I implemented a sort of automatic registration test, which I did a pull request for.

**Comment by watkipet** [comment]
Thanks, Michael. I pulled it. I updated the README to include building man pages as well (cmake ../src -> cmake ..). I'll have to get familiar with Travis CI.
